src/components/PageFooter.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {
+  Footer,
+  IconFacebook,
+  IconInstagram,
+  IconTwitter,
+  IconYoutube
+} from 'hds-react';
+
+/**
+ * NOTE: This is currently a placeholder footer. To be replaced by updated Footer component in HDS 3.0
+ */
+
+const PageFooter = () => (
+  <Footer title="Pysäköinnin Sähköinen Asiointi">
```

Comment:
```suggestion
  <Footer title="Pysäköinnin asiointi">
```
Let's use the same name as elsewhere